src/Microsoft.TestPlatform.CrossPlatEngine/Hosting/DefaultTestHostManager.cs

```diff
@@ -220,9 +221,10 @@ public IEnumerable<string> GetTestPlatformExtensions(IEnumerable<string> sources
             return Enumerable.Empty<string>();
         }
 
-        public bool CanExecuteCurrentRunConfiguration(RunConfiguration runConfiguration)
+        public bool CanExecuteCurrentRunConfiguration(string runConfiguration)
```

Comment:
Nit: 
Add XML comments for public methods.
Should we rename the arg to settingsXml just to make it intuitive ?
Prefer using var instead.